### PR TITLE
Need to get this merged to master, and published on repo.enonic.com

### PR DESCRIPTION
**Pull request by sigdestad**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Gradle Project settings
 projectName = hmdb
-version=2.2.0-SNAPSHOT
+version=2.3.0
```
Comment (anatol-sialitski) [inline gradle.properties:3]: According to release [procedure](https://enonic.atlassian.net/wiki/spaces/~419371546/pages/661389314/How+to+release+lib+site+or+app) version should be changed directly in master branch.

Comment (sigdestad) [reply gradle.properties:3]: Isn't this solved when merging?

Comment (sigdestad) [reply gradle.properties:3]: Same for me how it is done, can you fix it?

Comment (anatol-sialitski) [reply gradle.properties:3]: Sure, I will do it

**Comment by anatol-sialitski** [comment]
It was done in the separate [pr](https://github.com/enonic/app-hmdb/pull/24/files)

**Pull request closed without merging**